### PR TITLE
#93: Add svelte ssr example

### PR DESCRIPTION
**Pull request by Deadik**
Add simple example with svelte ssr

### CODE REVIEW

File: examples/svelte-ssr/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+require('svelte/register')
```
Comment (talentlessguy) [inline examples/svelte-ssr/index.js:2]: examples must use `import` / `export` syntax

File: examples/svelte-ssr/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "tinyhttp-svelte-ssr",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```
Comment (talentlessguy) [inline examples/svelte-ssr/package.json:5]: you forgot to put `"type": "module"`

**Review by talentlessguy — CHANGES_REQUESTED** [review]

File: examples/svelte-ssr/index.js (hunk 1 above)
Comment (Deadik) [reply examples/svelte-ssr/index.js:2]: Svelte has some problems with export esm `import 'svelte/register'` or `import 'svelte/compiler'` throw "module not found".
Haven’t come up with a solution yet.

Comment (talentlessguy) [reply examples/svelte-ssr/index.js:2]: gonna try it locally and see if it works

Comment (talentlessguy) [reply examples/svelte-ssr/index.js:2]: it works if you specify a file extension, e.g.:

```js
import 'svelte/register.js'
```

**Comment by Deadik** [comment]
#decline

**Comment by talentlessguy** [comment]
@Deadik did something happen? you closed the pr for some reason :c

**Comment by Deadik** [comment]
I'm not found working solution and can't spend more time

**Comment by talentlessguy** [comment]
@Deadik specifying the file extension worked

but if you have no time, thanks for your effort :D 

someone else might get it working

**Comment by Deadik** [comment]
specifying the file extension worked, but other my code with import not worked, should be changed to directly using svelte API for compile and render. A don't find how do this

**Pull request closed without merging**